TrackletAlgorithm/MatchCalculator.h

```diff
@@ -311,6 +311,34 @@ void MatchCalculator(BXType bx,
   bool read7 = false;
   bool read8 = false;
 
+  static const uint8_t shift_L1L2 = 0;
+  static const uint8_t shift_L2L3 = shift_L1L2 + 1;
+  static const uint8_t shift_L3L4 = shift_L2L3 + 1;
+  static const uint8_t shift_L4L5 = shift_L3L4 + 1;
+  static const uint8_t shift_L5L6 = shift_L4L5 + 1;
+  static const uint8_t shift_D1D2 = shift_L5L6 + 1;
+  static const uint8_t shift_D3D4 = shift_D1D2 + 1;
+  static const uint8_t shift_L1D1 = shift_D3D4 + 1;
+
+  static const uint16_t mask_L1L2 = 1 << shift_L1L2;
+  static const uint16_t mask_L2L3 = 1 << shift_L2L3;
+  static const uint16_t mask_L3L4 = 1 << shift_L3L4;
+  static const uint16_t mask_L4L5 = 1 << shift_L4L5;
+  static const uint16_t mask_L5L6 = 1 << shift_L5L6;
+  static const uint16_t mask_D1D2 = 1 << shift_D1D2;
+  static const uint16_t mask_D3D4 = 1 << shift_D3D4;
+  static const uint16_t mask_L1D1 = 1 << shift_L1D1;
+ 
+  // MC_L3PHIC mask {1: on, 0: off}
+  static const uint16_t FML1L2 = 1 << shift_L1L2;
+  static const uint16_t FML2L3 = 0 << shift_L2L3;
```

Comment:
what does it mean to shift a zero? I am confused. And are these static const or constexpr?